boards/ikea-tradfri/include/periph_conf.h

```diff
@@ -38,6 +38,13 @@ extern "C" {
  * @name    Clock configuration
  * @{
  */
+#ifndef CMU_HFXOINIT
+#define CMU_HFXOINIT        CMU_HFXOINIT_DEFAULT
```

Comment:
here and below: why define the default values for each board? Wouldn't it be much cleaner (and better scalable) to define the default values in `cpu/efm32_common/cpu.c`:
```c
#ifndef CMU_xx
#define CMU_xx   CMU_xx_DEFAULT
#endif
```
This way a board would only need to define these values if the board actually **does not** use the default values...

Reply:
I can also place them inline, for instance:

```c
#ifdef _SILICON_LABS_32B_SERIES_1
static void dcdc_init(void)
{
#ifdef EMU_DCDCINIT
    EMU_DCDCInit_TypeDef init_dcdc = EMU_DCDCINIT;
#else
    EMU_DCDCInit_TypeDef init_dcdc = EMU_DCDCINIT_DEFAULT;
#endif

    EMU_DCDCInit(&init_dcdc);
}
#endif
```

Added benefit: `cpu.c` is more to the point. What would be your preference?

Reply:
I personally prefer not to inline these things and have ifdef blocks in the beginning of the file. This makes the code much more readable.